crates/typst-cli/src/eval.rs

```diff
@@ -0,0 +1,148 @@
+use crate::{
+    args::{EvalCommand, FileInput, StringInput, SyntaxMode, Target},
+    compile::print_diagnostics,
+    set_failed,
+    world::{SystemWorld, decode_utf8, read_from_stdin},
+};
+use comemo::Track;
+use ecow::{EcoString, eco_format};
+use typst::{
+    World,
+    diag::{HintedStrResult, SourceResult, Warned},
+    engine::Sink,
+    foundations::{Binding, Context, Scope, StyleChain, Value},
+    introspection::Introspector,
+    layout::PagedDocument,
+    syntax::Span,
+};
+use typst_eval::eval_string;
+use typst_html::HtmlDocument;
```

Comment:
could you flatten these imports to make them consistent with the code base's style?